MainRepository/Program.cs

```diff
@@ -6,24 +6,21 @@
     if (input.ToLower() == "e") break;
     if (int.TryParse(input, out int n) && n > 0)
     {
-        bool result = ArmStrong(n);
-        if (!result) Console.WriteLine($"The given {n} number is not a armstrong number");
-        else Console.WriteLine($"The given {n} number is armstrong number");
+        bool result = Armstrong(n);
+        if (result) Console.WriteLine($"The given {n} number is armstrong number");
+        else Console.WriteLine($"The given {n} number is not a armstrong number");
     }
     else Console.WriteLine("Enter the valid input");
-    static bool ArmStrong(int n)
+
+    static bool Armstrong(int n)
     {
-        int sum = 0, temp = n, digits = 0, temp1 = n;
+        int sum = 0, temp = n;
+        int digits = n.ToString().Length;

```

Comment:
Instead of converting again int to string, you can pass the length param in Armstrong (int n, int len) => Armstrong (n, imput.Length).